docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -254,124 +223,121 @@ Follow these steps:
    git submodule update --init --recursive
    ```
 
-5. Build PyTorch for ROCm.
+5. Build PyTorch for ROCm. The default build directory is `rocm/pytorch:latest-base`.
 
    :::{note}
-   By default in the `rocm/pytorch:latest-base`, PyTorch builds for these
-   architectures simultaneously:
-   - gfx900
-   - gfx906
-   - gfx908
-   - gfx90a
-   - gfx1030
-   :::
+   PyTorch builds simultaneously for the following architectures:
```

Comment:
By default in the `rocm/pytorch:latest-base` image PyTorch builds simultaneously for the following architectures